index.html

```diff
@@ -48,5 +51,187 @@ <h2 class="prof">I’m a Software Developer</h2>
         </div>
     </div>
 </section>
+
+
+<!-- Recent Works -->
+
+<section id="works-section">
+
+
+  
+  <h3 class="hr">My Recent Work <hr> </h3>
+  
+
+
+<div class="box-container">
+  <div class="left-section">
+    <img src="./img/main (1).png" alt="Recent Project">
+    <div  class="first-desc">
+      <h3>Multi-Post Stories</h3>
+      <P>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</P>
+      <ul>
+        <li>CSS</li>
+        <li>HTML</li>
+        <li>Ruby</li>
+        <li>React</li>
+      </ul>
+      <button type="button">See project</button>
```

Comment:
- [ ] Please there is some issue with your design on a slightly larger screen ( width greater than 375px). Kindly fix that, below is the screenshot representing that 👇👇👇 Let's make your application responsive, won't we? 🤗🤗
Let's make sure that all items are on top of each other on the mobile device regardless of the width 😊 

![Screenshot from 2023-06-03 07-47-03](https://github.com/Xghayor/Portfolio/assets/10439283/db7cb51c-913f-4039-b266-ae79df5763b1)